JuiceMaker/JuiceMaker/Controller/JuiceOrderViewController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  JuiceMaker - ViewController.swift
+//  Created by yagom. 
+//  Copyright © yagom academy. All rights reserved.
+// 
+
+import UIKit
+
+class JuiceOrderViewController: UIViewController {
+    @IBOutlet var fruitStockLabels: [UILabel] = []
+    let juiceMaker = JuiceMaker(fruitStore: FruitStore(fruitInventory: [
+        .strawberry: 10,
+        .banana: 10,
+        .pineapple: 10,
+        .kiwi: 10,
+        .mango: 10
+    ]))
```

Comment:
```suggestion
    let juiceMaker = JuiceMaker(fruitStore: FruitStore(fruitInventory: [.strawberry: 10,
                                                                        .banana: 10,
                                                                        .pineapple: 10,
                                                                        .kiwi: 10,
                                                                        .mango: 10]))
```

저는 이러한 컨벤션으로 작성하는 것이 가독성에 좋을 것 같습니다.

그리고 `FruitStore`를 따로 프로퍼티로 선언하지 않고 `JuiceMaker.init`에서 초기화해 주입하는 이유가 있을까요?

Reply:
> 그리고 FruitStore를 따로 프로퍼티로 선언하지 않고 JuiceMaker.init에서 초기화해 주입하는 이유가 있을까요?

처음에는 특별한 이유 없이 `FruitStore`를 따로 프로퍼티로 뺴지 않고, `JuiceMaker.init`에서 초기화를 해줬습니다. 그래서 이렇게 코드를 작성한 이유를 곰곰이 생각을 한 뒤 내린 결론은 `FruitStore` 인스턴스를 은닉화를 하기 위해서입니다. `JuiceOrderViewController`에서는 `FruitStore`에 대한 인스턴스 정보를 직접적으로 알 필요가 없다고 생각합니다. 만약 각 과일의 현재 재고와 전체 과일의 재고를 알고 싶으면 `JuiceMaker` 내에 캡슐화된 메서드를 호출해서 해당 정보들을 알 수 있다고 생각했습니다. 또한, `JuiceOrderViewController`에서 `FruitStore`를 프로퍼티로 빼게 된다면 `JuiceMaker`에서 `FruitStore` 인스턴스를 은닉화 및 캡슐화한 이유가 없어진다고 생각했습니다.(외부(`JuiceOrderViewController`)에서 `FruitStore` 인스턴스에 접근할 수 있기 때문에)

Reply:
건디가 추천해주신 컨벤션이 훨씬 가독성에 좋은 것 같아서 적용 완료했습니다.